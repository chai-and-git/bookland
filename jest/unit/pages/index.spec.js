import { shallowMount, createLocalVue } from '@vue/test-utils';
import VueMeta from 'vue-meta';
import index from '@/pages/index.vue';

const localVue = createLocalVue();
localVue.use(VueMeta, { keyName: 'head' });

const indexWrapper = shallowMount(index, {
  localVue,
  stubs: {
    NuxtLink: true
  },
});

describe('index.vue', () => {
  it('is a Vue instance', () => {
      const wrapper = indexWrapper;
      expect(wrapper.vm).toBeTruthy();
  })

  it('is a Vue instance', () => {
    const wrapper = indexWrapper;
    expect(wrapper.html()).toMatchSnapshot();
  })
})