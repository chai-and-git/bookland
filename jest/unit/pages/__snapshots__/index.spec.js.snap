// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.vue is a Vue instance 1`] = `
"<div class=\\"home\\">
  <div class=\\"description\\">
    <h2 class=\\"title\\">Welcome to the land of books...</h2>
    <h3 class=\\"subtitle\\">Grab a cup of coffee and savour upon your favorite author's book!</h3>
  </div>
</div>"
`;
